docs/overview.asciidoc

```diff
@@ -0,0 +1,39 @@
+[[overview]]
+== Overview 
+
+--
```

Comment:
What does the `--` do?

Reply:
It's a [part intro](https://github.com/elastic/docs/blob/master/README.asciidoc#part-intro)

Reply:
Shouldn't it then have a `[partintro]` line just above the `--`?

Reply:
Reading the asciidoc docs, it seems that this is only needed for parts which are level 0 (`=`). This is probably also why it's not complaining about the missing `[partintro]`